packages/shared/src/services/SessionResilienceServiceSimple.ts

```
@@ -0,0 +1,97 @@
+        lastActivity: Date.now(),
+        createdAt: Date.now(),
+        userId: "offline-user",
+        metadata: {
+          ...params.metadata,
+          // Note: Adding custom properties that extend SessionMetadata
+        } as any,
+        version: 1
+      };
+      
+      return offlineSession;
+    });
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)